packages/delisp-core/src/compiler.ts

```diff
@@ -116,24 +117,22 @@ function compileTopLevel(syntax: Syntax, env: Environment): JSAST {
   }
 }
 
-export function compileModule(syntax: Syntax): JSAST {
+export function compileModule(syntaxes: Syntax[]): JSAST {
```

Comment:
Let's make this `module: Module` instead?  I know it is used from `compileToString`, but see the comment below.